### ISSUE TITLE
Nested Virtualization

### ISSUE THREAD

**Issue by garloff** [SovereignCloudStack/Operational-Docs#19]
We currently don't have a way to express in the flavor name whether or not the flavors support nested virtualization.
We should expect nested virt to be default-on and have a modifier to show that it's off?

**Comment by tibeer** [comment]
Agreed. Identifier for disabled nested virtualization: "-nonested"?
